framework-plugins/lisk-framework-forger-plugin/test/integration/specs/forger_info.spec.ts

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright © 2020 Lisk Foundation
```

Comment:
why is this integration not functional? it seems it's starting the application and calling the endpoint.

Reply:
since it was related to event, i thought its more appropriate for integration.

Reply:
As using http-api to post transactions, means its running a full flow of external activities to get the application in a state to test assertion, its should be moved to functional tests.